Pocket/pocket-ios
# Alternate Icons - Set Pride icon as default

## Summary
* This PR sets the Pride icon as default
* It also adds the `Automatic` identifier for analytics when the default icon is selected

## References 
* Branch name

## Implementation Details
* See summary

## Test Steps
* Install any version (e.g. App Store)
* Build/run this branch over the existing installation
* Make sure the icon changes to the pride icon
* Delete and build/run again
* Make sure the icon is the pride icon
* Select a different icon from the icon switcher
* Build/run again and make sure the selection is preserved

## PR Checklist:
